spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,9 +323,46 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response(app_id: nil, platform: nil, review_id: nil, response: "")
```

Comment:
i am more a fan of named parameters, makes it easier to refactor later on. (but non-optional named params introduced some problems on older ruby versions, on prev. changes)

but you are right  - @KrauseFx @mfurtak can you jump in here what is the preferred way?